Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -722,6 +722,10 @@
 #define ENABLE_WEB_CODECS 1
 #endif
 
+#if !defined(ENABLE_WK_WEB_EXTENSIONS) && !PLATFORM(WATCHOS) && !PLATFORM(APPLETV)
```

Comment:
I think it might be better to list the platforms where it is supported rather than the platforms where it is not supported.